source/newpmcmember.md

```diff
@@ -0,0 +1,172 @@
+committed to the project and graceful toward users and other developers.
+
+Don't wait too long before proposing and don't be too hasty. There is a
+trade-off and something about timeliness. A point is reached where it
+becomes obvious that we should invite them. This encourages them and keeps
+them enthusiastic. If we leave it too long, then we risk them becoming
+disillusioned.
+
+On the `private@` list we can each say exactly what we feel about each person,
+with no holds barred. Keep the discussion concise. The praise part can
+be done later in public. Keep in mind, however, that if the member becomes
+a PMC member later, they will have access to this discussion.
+
+Let the Vote thread run for one week.
+
+A positive result is achieved by **Consensus Approval**: at least 3 +1
+votes and no vetoes.
+
+Any veto must be accompanied by reasoning and the vetoer must be prepared to defend it.
+Other members can attempt to encourage them to change their mind.
+
+New PMC members can be either quiet or active as they choose. If we find
+that certain people lapse and don't ever contribute, then the project can take steps
+to retire them.
+
+After a positive result, record the result on the PMC list with a `[RESULT][VOTE]` subject
+and then invite the candidate. We give candidates a chance to decline PMC membership in private.
+They can post a reply to the PMC mailing list.
+
+After we reach a decision on the `private@` list, and after the steps above, we
+announce the new PMC member on the `dev` list. We can then each follow up with
+our praise in public.
+
+Other notes about the process are available on the main [Apache site](https://www.apache.org/dev/pmc.html#newpmcmember).
+
+<a name="NewCommitter-EmailTemplates"></a>
+## Email Templates
+
+### PMC member Vote Template
+This is the email to commence a vote for a new PMC member.
+
+    ------------------------------------------------------------
+    To: private@[PROJECT].apache.org
+    Subject: [VOTE] New PMC member candidate: Jo Bloggs
+
+    [ add the reasons behind your nomination here ]
+
+    Voting ends one week from today, i.e. midnight UTC on YYYY-MM-DD
+    https://www.timeanddate.com/counters/customcounter.html?year=YYYY&month=MM&day=DD
+
+    See voting guidelines at
+    https://community.apache.org/newpmcmember.html
+
+    ------------------------------------------------------------
+
+### Close Vote
+This email ends the vote and reports the result to the project.
+
+    ------------------------------------------------------------
+    To: private@[PROJECT].a.o
+    Subject: [RESULT] [VOTE] New PMC member candidate: Jo Bloggs
+
+    The vote has now closed. The results are:
+
+    Binding Votes:
+
+    +1 [TOTAL BINDING +1 VOTES]
+     0 [TOTAL BINDING +0/-0 VOTES]
+    -1 [TOTAL BINDING -1 VOTES]
+
+    The vote is ***successful/not successful***
+
+### Notify Board of new PMC member
+See [new PMC member](https://www.apache.org/dev/pmc.html#newpmcmember)
+
```

Comment:
This is no longer required, as long as the Whimsy tool is used to add the new PMC member to the roster.